.github/workflows/main-build.yml

```diff
@@ -31,8 +31,9 @@ jobs:
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
             type=ref,event=tag,suffix=-rc
-            type=sha,suffix=-rc
-            type=sha,format=long,suffix=-rc
+            type=sha,suffix=-rc,format=short
+            type=sha,format=short,suffix=-rc
```

Comment:
This line seems to contain the same parameters as previous one, just different order. Is that intended?